src/index.ts

```
@@ -0,0 +1,369 @@
+    cluster.setupPrimary(settings);
+  }
+
+  const disconnects: Record<number, string> = {};
+  let disconnectCount = 0;
+  let unexpectedCount = 0;
+
+  cluster.on('disconnect', function(worker) {
+    const disableRefork = getDisableRefork(worker);
+    const log = console[ disableRefork ? 'info' : 'error'];
+    disconnectCount++;
+    const isDead = worker.isDead && worker.isDead();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)